README.md

```diff
@@ -26,12 +26,14 @@ To run your own development testing version of the API, follow these steps:
 ### Setup the running environment
 
 You can use the `docker-compose` which is more than sufficient for testing:
-*Note: this guide is meant for linux, if you're on windows, search how to run docker-compose there*
+_Note: this guide is meant for linux, if you're on windows, search how to run docker-compose there_
 
 1. Install docker, specifically `docker-compose`. (I won't detail this, for Arch Linux it's `pacman -S docker-compose`)
 2. Go to the into the root of the repository. (`cd /path/to/rickchurch`)
-3. Simply run `docker-compose up` (you might need `sudo`)
-4. The API is now running at `localhost:8000`, and if you need to access the postgresql database, it's at `localhost:5000`
+3. Configure the app using environment variables or a .env file in the root of the repository.
```

Comment:
The environmental variables are described specifically in the section above, there's no need for this line considering it's above.

Though you should provide a richer detail about the environmental variables that may need to be set, mentioning `constants` makes a lot of sense and so only the needed env. variables can be documented and if user wants something else, he can look at the constants, but it still belongs to the environment section not here.

Reply:
Oops, I didn't read that part